service/awsconfig/cluster_framework.go

```diff
@@ -25,11 +25,7 @@ import (
 	v6cloudconfig "github.com/giantswarm/aws-operator/service/awsconfig/v6/cloudconfig"
 )
 
-const (
-	AWSConfigCleanupFinalizer = "aws-operator.giantswarm.io/custom-object-cleanup"
-)
```

Comment:
This is left over from the TPR migration. We do not need this at all. Just boyscouting.